charts/apisix-ingress-controller/values.yaml

```diff
@@ -122,6 +122,8 @@ config:
     enableGatewayAPI: false
   # APISIX related configurations.
   apisix:
+    # Enabling this value, overrides serviceName and serviceNamespace.
+    # serviceFullname: "apisix-admin"
```

Comment:
I think we should can set up an example here. e.g. `apisix-admin.apisix.svc.local`

Reply:
Done.